src/GraphQL/DuplicateElementMutation.php

```diff
@@ -56,6 +56,11 @@ public function resolve($object, array $args, $context, ResolveInfo $info)
                 "The current user has insufficient permission to edit ElementalArea: $areaID"
             );
         }
+        if (!$element->canCreate($content['currentUser'])) {
```

Comment:
Seems less than ideal fixing this only in elemental rather than in the graphql module itself - is there a way we could fix this there instead?

Reply:
I've had a look withing the graphql module and I cannot find anything explicitly for handling duplication there, so I guess it makes sense to do this here